packages/editor/src/components/post-preview-button/index.js

```diff
@@ -91,6 +91,19 @@ function writeInterstitialMessage( targetDocument ) {
 	targetDocument.close();
 }
 
+const downArrow = (
+	<SVG width="18" height="18" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 18 18">
+		<Polygon points="9,13.5 14.7,7.9 13.2,6.5 9,10.7 4.8,6.5 3.3,7.9 " />
+	</SVG>
+);
+
+const checkIcon = (
+	<SVG xmlns="http://www.w3.org/2000/svg" width="24" height="24">
+		<Path fill="none" d="M0 0h24v24H0z" />
+		<Path d="M9 16.2L4.8 12l-1.4 1.4L9 19 21 7l-1.4-1.4L9 16.2z" />
+	</SVG>
+);
```

Comment:
Not related to this PR, but noticed we have inline SVGs all over the codebase now. Might be nice to consolidate them into a single file/package so that we don't accidentally end up with duplicated code in the bundle.